examples/tomo/ray_trafo_parallel_2d_complex.py

```diff
@@ -0,0 +1,45 @@
+"""Example computing the 2d parallel beam ray transform of a complex phantom.
+
+This example demonstrates that complex-valued functions work for both
+forward and back projections by application to real and imaginary parts
+separately.
```

Comment:
The "separately" part here implies that this is in some sense done by the user, or in a way visible to the user. I'd simply say:

"This example demonstrates that forward and back projections work with complex-valued functions."""